git-hub

```diff
@@ -341,6 +341,14 @@ class Config:
     def __init__(self):
         self.username = git_config('username', getpass.getuser())
         self.oauthtoken = git_config('oauthtoken')
+        self.oauthtokencmd = git_config('oauthtokencmd')
+        if self.oauthtokencmd is not None:
+            try:
+                self.oauthtoken = subprocess.check_output(
+                        self.oauthtokencmd.split()).strip()
```

Comment:
Use `shlex.split()` for command-line splitting.
Or, better, pass the whole command line to shell:
```python
subprocess.check_output(self.oauthtokencmd, shell=True)
```

Reply:
Done